src/scene/graph-node.js

```diff
@@ -876,11 +876,9 @@ Object.assign(pc, function () {
         },
 
         _dirtifyLocal: function () {
-            if (!this._dirtyLocal) {
```

Comment:
This check is always done at the call sites. Checking again here is unnecessary.